### DISCUSSION TITLE
Module was compiled with an incompatible version of Kotlin

### DISCUSSION THREAD

**Discussion by manishoo** [software-mansion/react-native-screens#1076] [Q&A] [answered]
After upgrading to 3.6.0, I'm faced with the following error. Can somebody give me a clue what I should do?

`/Users/manishoo/.gradle/caches/transforms-3/a0fd1432ff5b5a7ad399c3ce1730c317/transformed/jetified-kotlin-stdlib-common-1.5.20.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.5.1, expected version is 1.1.16.`

![image](https://user-images.githubusercontent.com/16239413/130744047-b048a55c-ac09-4725-9dc4-11189d907e67.png)

**Comment by manishoo** [comment]
I've tried upgrading gradle to `gradle-6.9-all.zip` and changing `kotlinVersion` in android/build.gradle to `kotlinVersion = '1.3.61'` and `kotlinVersion = '1.5.1'`

**Comment by markrickert** [comment]
In my case, i had to upgrade `kotlin-gradle-plugin` to version 1.5.20 in the `/android/build.gradle`

```
buildscript {
    dependencies {
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.20"
    }
}
```

Didn't have to specify a kotlin version because each library should do that on its own.

**Comment by manishoo** [comment, marked as answer]
I upgraded Gradle to 6.9 and that solved the problem
